UPGRADE-6.2.md

```diff
@@ -19,6 +19,26 @@ DependencyInjection
  * Change the signature of `ContainerAwareInterface::setContainer()` to `setContainer(?ContainerInterface)`
  * Deprecate calling `ContainerAwareTrait::setContainer()` without arguments
  * Deprecate using numeric parameter names
+ * Deprecate the injected `$env` variable in the scope of php config files, use `#[When]` or inject it into the closure
```

Comment:
I'd remove the deprecation, there is no need to remove the variable.

Reply:
Done